example/advection/parthinput.advection

```diff
@@ -83,6 +83,11 @@ file_type = hst
 dt = 0.05
 
 <parthenon/output4>
+file_type = hst
+dt = 0.1
+packages = advection_app
+
+<parthenon/output5>
 file_type = ascent
 dt = -0.05   # soft disabled by default, as Ascent is an optional dependency
```

Comment:
I think this is what's causing the failed regression test as ascent outputs are manually enabled in `.github/workflows/ci-extended.yml` via `parthenon/output4/dt=0.05`

Reply:
🚨 thank you for noticing this @pgrete! 

I also modified `.github/workflows/ci-extended.yml` to give an error message if `test` fails.